src/app.d.ts

```diff
@@ -71,12 +71,12 @@ declare global {
           whileLoop: string[];
         };
         modifyOutput?: (
-          output: SeqJson | any,
+          output: string | any,
```

Comment:
Can we instead import the SeqJson type into the file?

Reply:
Oh...didn't read the PR description. So is the new correct type that `output` is strictly a `string`?

Reply:
I changed this because the `SeqJson` type is no longer correct; the default value of `output` is a stringified JSON.

Reply:
Didn't refresh to see your second comment but yes, `output` is now a string according to the default `toOutputFormat`. However, a mission adaptation _could_ provide their own `toOutputFormat` which returns some other type that's then ingested by `modifyOutput` (not sure what that enables in their workflow, but it's mechanically possible AFAIK).

Reply:
Oh sike you're right, yes this can all be tightened down to just be `string`!

Reply:
Thanks!